### PR TITLE
Update blog pages and add initial styling

### PR DESCRIPTION
**Pull request by kennyalmendral**

### CODE REVIEW

File: web/wp-content/themes/dctx/single.php [hunk 1]
```diff
@@ -5,9 +5,21 @@
...
 				<?php dctx_post_meta(); ?>
 
+				<?php if (has_post_thumbnail()): ?>
+					<div class="featured-image">
+						<?php the_post_thumbnail('full'); ?>
+
+						<?php if (get_post( get_post_thumbnail_id())->post_content): ?>
+							 <div class="featured-image-caption">
+								  <?php echo wp_kses_post(get_post( get_post_thumbnail_id() )->post_content ); ?>
+							 </div>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :warning: **Warning**: Found precision alignment of 1 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).

File: web/wp-content/themes/dctx/single.php [hunk 2]
```diff
@@ -5,9 +5,21 @@
...
+				<h1 class="post-title"><?php the_title(); ?></h1>
 				<?php dctx_post_meta(); ?>
 
+				<?php if (has_post_thumbnail()): ?>
+					<div class="featured-image">
+						<?php the_post_thumbnail('full'); ?>
+
+						<?php if (get_post( get_post_thumbnail_id())->post_content): ?>
+							 <div class="featured-image-caption">
+								  <?php echo wp_kses_post(get_post( get_post_thumbnail_id() )->post_content ); ?>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :warning: **Warning**: Found precision alignment of 2 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).

File: web/wp-content/themes/dctx/single.php [hunk 3]
```diff
@@ -5,9 +5,21 @@
...
-				<h1><?php the_title(); ?></h1>
+				<h1 class="post-title"><?php the_title(); ?></h1>
 				<?php dctx_post_meta(); ?>
 
+				<?php if (has_post_thumbnail()): ?>
+					<div class="featured-image">
+						<?php the_post_thumbnail('full'); ?>
+
+						<?php if (get_post( get_post_thumbnail_id())->post_content): ?>
+							 <div class="featured-image-caption">
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :warning: **Warning**: Found precision alignment of 1 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).

File: web/wp-content/themes/dctx/single.php [hunk 4]
```diff
@@ -5,9 +5,21 @@
...
 			<?php while (have_posts()) : the_post(); ?>
-				<h1><?php the_title(); ?></h1>
+				<h1 class="post-title"><?php the_title(); ?></h1>
 				<?php dctx_post_meta(); ?>
 
+				<?php if (has_post_thumbnail()): ?>
+					<div class="featured-image">
+						<?php the_post_thumbnail('full'); ?>
+
+						<?php if (get_post( get_post_thumbnail_id())->post_content): ?>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: Space between opening control structure and T_COLON is required (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBetweenStructureColon*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: Expected 1 space(s) after closing parenthesis; found 0 (*Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: No space before closing parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeCloseParenthesis*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: No space after opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis*).

File: web/wp-content/themes/dctx/single.php [hunk 5]
```diff
@@ -5,9 +5,21 @@
 
 		<div class="main-content">
 			<?php while (have_posts()) : the_post(); ?>
-				<h1><?php the_title(); ?></h1>
+				<h1 class="post-title"><?php the_title(); ?></h1>
 				<?php dctx_post_meta(); ?>
 
+				<?php if (has_post_thumbnail()): ?>
+					<div class="featured-image">
+						<?php the_post_thumbnail('full'); ?>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

File: web/wp-content/themes/dctx/single.php [hunk 6]
```diff
@@ -5,9 +5,21 @@
 
 		<div class="main-content">
 			<?php while (have_posts()) : the_post(); ?>
-				<h1><?php the_title(); ?></h1>
+				<h1 class="post-title"><?php the_title(); ?></h1>
 				<?php dctx_post_meta(); ?>
 
+				<?php if (has_post_thumbnail()): ?>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: Space between opening control structure and T_COLON is required (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBetweenStructureColon*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: Expected 1 space(s) after closing parenthesis; found 0 (*Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: No space before closing parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeCloseParenthesis*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: No space after opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis*).

File: web/wp-content/themes/dctx/single.php [hunk 7]
```diff
@@ -5,9 +5,21 @@
 
 		<div class="main-content">
 			<?php while (have_posts()) : the_post(); ?>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: Newline required after opening brace (*Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: No space before closing parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeCloseParenthesis*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php]: :no_entry_sign: **Error**: No space after opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis*).

File: web/wp-content/themes/dctx/index.php [hunk 8]
```diff
@@ -1,21 +1,25 @@
...
+	<h1 class="page-title">Blog</h1>
+
 	<div class="container clearfix">
 
 		<div class="main-content">
 
 			<?php if (have_posts()) : ?>
-				<ul class="posts">
+				<div class="posts">
 					<?php while (have_posts()) : the_post(); ?>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/index.php]: :no_entry_sign: **Error**: Newline required after opening brace (*Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/index.php]: :no_entry_sign: **Error**: No space before closing parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeCloseParenthesis*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/index.php]: :no_entry_sign: **Error**: No space after opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis*).

File: web/wp-content/themes/dctx/index.php [hunk 9]
```diff
@@ -1,21 +1,25 @@
 <?php get_header(); ?>
 
 <div id="main">
+	<h1 class="page-title">Blog</h1>
+
 	<div class="container clearfix">
 
 		<div class="main-content">
 
 			<?php if (have_posts()) : ?>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/index.php]: :no_entry_sign: **Error**: No space before closing parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeCloseParenthesis*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/index.php]: :no_entry_sign: **Error**: No space after opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis*).

File: web/wp-content/themes/dctx/index.php [hunk 10]
```diff
@@ -1,21 +1,25 @@
 <?php get_header(); ?>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/index.php]: :no_entry_sign: **Error**: Missing file doc comment (*Squiz.Commenting.FileComment.Missing*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/index.php]: :no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-ntyjwn.php, but found phpcs-scan-NTYjwn.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).

File: web/wp-content/themes/dctx/archive.php [hunk 11]
```diff
@@ -1,18 +1,20 @@
...
+	<?php the_archive_title('<h1 class="page-title">', '</h1>'); ?>
+
 	<div class="container clearfix">
 		
 		<div class="main-content">
-			<?php the_archive_title('<h1 class="page-title">', '</h1>'); ?>
-
 			<?php if (have_posts()) : ?>
 				<ul class="posts">
 					<?php while (have_posts()) : the_post(); ?>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/archive.php]: :no_entry_sign: **Error**: Newline required after opening brace (*Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/archive.php]: :no_entry_sign: **Error**: No space before closing parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeCloseParenthesis*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/archive.php]: :no_entry_sign: **Error**: No space after opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis*).

File: web/wp-content/themes/dctx/archive.php [hunk 12]
```diff
@@ -1,18 +1,20 @@
...
 
 <div id="main">
+	<?php the_archive_title('<h1 class="page-title">', '</h1>'); ?>
+
 	<div class="container clearfix">
 		
 		<div class="main-content">
-			<?php the_archive_title('<h1 class="page-title">', '</h1>'); ?>
-
 			<?php if (have_posts()) : ?>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/archive.php]: :no_entry_sign: **Error**: No space before closing parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeCloseParenthesis*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/archive.php]: :no_entry_sign: **Error**: No space after opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis*).

File: web/wp-content/themes/dctx/archive.php [hunk 13]
```diff
@@ -1,18 +1,20 @@
 <?php get_header(); ?>
 
 <div id="main">
+	<?php the_archive_title('<h1 class="page-title">', '</h1>'); ?>
+
 	<div class="container clearfix">
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/archive.php]: :warning: **Warning**: Found precision alignment of 1 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).

File: web/wp-content/themes/dctx/archive.php [hunk 14]
```diff
@@ -1,18 +1,20 @@
 <?php get_header(); ?>
 
 <div id="main">
+	<?php the_archive_title('<h1 class="page-title">', '</h1>'); ?>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/archive.php]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/archive.php]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

File: web/wp-content/themes/dctx/archive.php [hunk 15]
```diff
@@ -1,18 +1,20 @@
 <?php get_header(); ?>
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/archive.php]: :no_entry_sign: **Error**: Missing file doc comment (*Squiz.Commenting.FileComment.Missing*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/archive.php]: :no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-k4yl3w.php, but found phpcs-scan-k4YL3w.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).

**Comment by carl-alberto** [comment]
You can put in a `// phpcs:disable` directive at the top of the file to ignore the checks @kennyalmendral, while @khleomix tries to revamp it on a later time

You can also set you local computer to do PHPCS using this https://github.com/WordPress/WordPress-Coding-Standards so you can make sure your code is compliant when pushing it

File: web/wp-content/themes/dctx/index.php (hunk 10 above)
Comment (wpugphil) [inline web/wp-content/themes/dctx/index.php:1]: :no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-19erlm.php, but found phpcs-scan-19ERlm.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).

File: web/wp-content/themes/dctx/archive.php [hunk 16]
```diff
@@ -1,18 +1,24 @@
-<?php get_header(); ?>
+<?php
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/archive.php:1]: :no_entry_sign: **Error**: Whitespace found at end of line (*Squiz.WhiteSpace.SuperfluousWhitespace.EndLine*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/archive.php:1]: :no_entry_sign: **Error**: Missing file doc comment (*Squiz.Commenting.FileComment.Missing*).

File: web/wp-content/themes/dctx/single.php [hunk 17]
```diff
@@ -1,13 +1,29 @@
-<?php get_header(); ?>
+<?php
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php:1]: :no_entry_sign: **Error**: Whitespace found at end of line (*Squiz.WhiteSpace.SuperfluousWhitespace.EndLine*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php:1]: :no_entry_sign: **Error**: Missing file doc comment (*Squiz.Commenting.FileComment.Missing*).

File: web/wp-content/themes/dctx/index.php [hunk 18]
```diff
@@ -1,21 +1,29 @@
-<?php get_header(); ?>
+<?php
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/index.php:1]: :no_entry_sign: **Error**: Whitespace found at end of line (*Squiz.WhiteSpace.SuperfluousWhitespace.EndLine*).

Comment (wpugphil) [inline web/wp-content/themes/dctx/index.php:1]: :no_entry_sign: **Error**: Missing file doc comment (*Squiz.Commenting.FileComment.Missing*).

File: web/wp-content/themes/dctx/single.php [hunk 19]
```diff
@@ -1,13 +1,29 @@
...
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/single.php:1]: :no_entry_sign: **Error**: Missing file doc comment (*Squiz.Commenting.FileComment.Missing*).

File: web/wp-content/themes/dctx/index.php [hunk 20]
```diff
@@ -1,21 +1,29 @@
...
```
Comment (wpugphil) [inline web/wp-content/themes/dctx/index.php:1]: :no_entry_sign: **Error**: Missing file doc comment (*Squiz.Commenting.FileComment.Missing*).

**Comment by carl-alberto** [comment]
Closes #19

**Pull request merged**